sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -119,8 +106,16 @@ public EventData(BinaryData body) {
     EventData(BinaryData body, SystemProperties systemProperties, Context context) {
         this.body = Objects.requireNonNull(body, "'body' cannot be null.");
         this.context = Objects.requireNonNull(context, "'context' cannot be null.");
-        this.systemProperties =  Objects.requireNonNull(systemProperties, "'systemProperties' cannot be null.");
-        this.properties = new HashMap<>();
+        this.systemProperties = Objects.requireNonNull(systemProperties, "'systemProperties' cannot be null.");
```

Comment:
@conniey:  How does this work if someone grabs the underlying AMQP annotated message, and changes some annotations?   Wouldn't `EventData` not reflect the current state?

Reply:
For context, I had to jump through some strange hoops to keep the behavior of `SystemProperties` consistent with the existing GA behavior and also handle the fact that the annotated message is fully mutable.

[AmqpSystemProperties](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpSystemProperties.cs)

Reply:
According to your instructions, fix in new version.